code/datums/elements/hatstacker.dm

```diff
@@ -0,0 +1,101 @@
+/**
+ * ### hat stacker element!
+ *
+ * Non bespoke element (1 in existence) that lets helmets stack hats on top of themselves!
+ * If someone wants to change this list to include stuff that fits on one thing and not on the mod helmets, convert this to a bespoke element,
+ * and make pre-defined lists to apply whenever we want.
+ * Will also be a good time to kill PLASMAMAN_HELMET_EXEMPT
+ */
+
+///Trait applied when a hat is currently stacked on another hat; used for tracking
+#define TRAIT_HATSTACKED 1
```

Comment:
Why is this trait `1` and not a string?

Also, put this ABOVE your documentation for the element itself

Reply:
Confusion mostly
Will do